Source/Blazorise/Components/Typography/DescriptionListDefinition.razor.cs

```diff
@@ -18,13 +19,32 @@ public partial class DescriptionListDefinition : BaseTypographyComponent
 
     #region Methods
 
+    /// <inheritdoc/>
+    protected override void OnInitialized()
+    {
+        ParentRowState?.NotifyColumnInitialized( this );
+
+        base.OnInitialized();
+    }
+
+    /// <inheritdoc/>
+    protected override void Dispose( bool disposing )
+    {
+        if ( disposing )
+        {
+            ParentRowState?.NotifyColumnRemoved( this );
+        }
+
+        base.Dispose( disposing );
+    }
```

Comment:
I feel like there's something missing here, doesn't the default Dispose template have something like `isDisposed`?

Reply:
It does, and we check it in the base class:

```
if ( !Disposed )
{
    Disposed = true;

    if ( disposing )
    {
        executeAfterRenderQueue?.Clear();

        ComponentDisposer.Dispose( this );
    }
```

Although the order of execution in the overridden method could be reversed, I don't think this is so wrong.

Reply:
kk